projects/2023TemplateWithVite/client/src/components/contact.jsx

```diff
@@ -0,0 +1,49 @@
+
+
+export default function Contact() {
+  const contact = {
+    first: "Cristina",
+    last: "Rodriguez",
+    avatar: "https://placekitten.com/g/200/200",
+    twitter: "yosola",
+    notes: "Some notes",
+    favorite: true,
+  };
+
+  return (
+    <div id="contact">
+      <div>
+        <img
+          key={contact.avatar}
```

Comment:
A `key` prop is not necessary unless you're in a loop OR are abusing the `key` prop to explicitly trigger re-renders (which doesn't look to be the case here)